src/frontend/src/main.js

```diff
@@ -58,13 +59,15 @@ async function setupGame() {
     "square",
     "square",
   ]);
-
   render(cameraControls, spotlightHelper);
   /* Example actions
   logInfo("Hello this is a fucking cat.");
   matchEndScene("loss");
   characterDeath(document.querySelector('[char-id="2"]'));
   */
+  document.querySelector(".end-turn-btn").addEventListener('click',() => {
+    Myloop(GameState);
```

Comment:
Why is a function called loop called when clicking?

Reply:
this is just the convinient function, it calls all the other function in the sequence from the flowchart it is just for testing and stuff.